src/core/types/State.ts

```diff
@@ -33,6 +34,7 @@ export interface RootState {
 
 export interface AppState {
   isInitialized: boolean;
+  servicesEnabled: Record<ServiceId, boolean>;
```

Comment:
nit: In terms of extensibility I would say that this could be a bit more than a boolean and also perhaps use the enum naming convention where it's singular instead of plural
```suggestion
  serviceStatus: Record<ServiceId, { enabled: boolean }>;
```
usage
```typescript
const { enabled: notifyEnabled } = app.serviceStatus['notify'];
```
No action needed, just a thought